Dockerfile

```diff
@@ -15,7 +15,7 @@ RUN mkdir /etc/service/cromwell && \
     apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D && \
     echo 'deb https://apt.dockerproject.org/repo ubuntu-trusty main' > /etc/apt/sources.list.d/docker.list && \
     apt-get update && \
-    apt-get install yq docker-engine=1.8.2-0~trusty && \
+    apt-get -yq install docker-engine=1.8.2-0~trusty && \
```

Comment:
Gotcha, `apt-get install -yq` is more common. Still, the other lines use `apt-get -yq …`, and the `apt-get` [man page](http://manpages.ubuntu.com/manpages/xenial/en/man8/apt-get.8.html) lists the command as:

> SYNOPSIS
       apt-get [-asqdyfmubV] [-o=config_string] [-c=config_file]
               [-t=target_release] [-a=architecture] {update | upgrade |
               dselect-upgrade | dist-upgrade |
               install pkg [{=pkg_version_number | /target_release}]...  |
               remove pkg...  | purge pkg...  |
               source pkg [{=pkg_version_number | /target_release}]...  |
               build-dep pkg [{=pkg_version_number | /target_release}]...  |
               download pkg [{=pkg_version_number | /target_release}]...  |
               check | clean | autoclean | autoremove | {-v | --version} |
               {-h | --help}}

So I'm leaving it unless we get an override 2nd vote before this pr gets another thumbsup.